system-tests/projects/e2e/cypress/e2e/cy_origin_error.cy.ts

```diff
@@ -37,8 +37,25 @@ describe('cy.origin errors', () => {
   })
 
   verify('failure when using dependency', this, {
-    line: 32,
-    column: 8,
+    line: 35,
+    column: 10,
+    message: 'Expected to find element',
+    stack: ['cy_origin_error.cy.ts'],
+    before () {
+      cy.visit('/primary_origin.html')
+    },
+    skipTitleValidation: true,
```

Comment:
```suggestion
    // Skip title validation here since the command is deep enough in the test that it does not show the command title
    skipTitleValidation: true,
```